### PR TITLE
Add index to call trace DB to speed up lookup by module

### PR DESCRIPTION
**Pull request by Feuermurmel**
Resolves https://github.com/Instagram/MonkeyType/issues/300.

### CODE REVIEW

File: monkeytype/db/sqlite.py [hunk 1]
```diff
@@ -21,19 +21,28 @@
...
+        CREATE INDEX IF NOT EXISTS {table}_module ON {table} (module);
+        """
+    ]
+
     with conn:
-        conn.execute(query)
+        for query in queries:
+            conn.execute(
+                query.format(table=table)
+            )
```
Comment (carljm) [inline monkeytype/db/sqlite.py:45]:
```suggestion
            conn.execute(query.format(table=table))
```

File: monkeytype/db/sqlite.py [hunk 2]
```diff
@@ -21,19 +21,28 @@
...
+          qualname    TEXT,
+          arg_types   TEXT,
+          return_type TEXT,
+          yield_type  TEXT);
+        """,
+        """
+        -- This index speeds up lookups of call traces of a single module
+        -- (see `make_query()`).
+        CREATE INDEX IF NOT EXISTS {table}_module ON {table} (module);
+        """
```
Comment (carljm) [inline monkeytype/db/sqlite.py:38]:
```suggestion
        """,
```

**Comment by carljm** [comment]
Hi @Feuermurmel, this looks great! It needs a couple minor tweaks (per my comments) to pass our autoformatter; unfortunately it looks like you didn't check the box that would allow me to make those updates to your PR myself, so are you willing to accept my suggestions so we can get this merged? Thank you!

**Review by carljm — CHANGES_REQUESTED** [review]
Putting it back in your queue for the formatting updates.

**Comment by Feuermurmel** [comment]
> Hi @Feuermurmel, this looks great! It needs a couple minor tweaks (per my comments) to pass our autoformatter

Same issue here: https://github.com/Instagram/MonkeyType/pull/302#issuecomment-1725279372

What's the expectation here? Should I only format the files that I touched and leave the rest as-is, and that will pass the checks?

**Comment by carljm** [comment]
Just format the files within `monkeytype/` directory, and that should also format only the files you touched. Or in this case, you can just accept my suggested edits above and that will also pass the formatter check.

**Comment by Feuermurmel** [comment]
It should be fixed now!

**Pull request merged**